.eslintrc.json

```diff
@@ -0,0 +1,69 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "installedESLint": true,
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true
+        }
+    },
+    "plugins": [
+        "mozilla"
+    ],
+    "globals": {
+        "Components": false,
+        "Services": false,
+        "Cc": false,
+        "Ci": false,
+        "VTTabDataStore": false,
+        "VerticalTabs": false,
```

Comment:
unfortunately any of the ones that were in the xml I cannot seem to exclude in the file. Eslint can't seem to read <-- -->  styly comments